src/pages/index.js

```diff
@@ -65,34 +166,65 @@ consts.profileEditButton.addEventListener("click", () => {
 consts.profileAddButton.addEventListener("click", () => {
   addCardModal.open();
 });
+consts.avatarContainer.addEventListener("click", () => {
+  avatarEditModal.open();
+});
 
-// validation
-
-const addCardValidator = new FormValidator(
-  consts.validatorConfig,
-  consts.addCardModalDiv,
-);
-
-addCardValidator.enableValidation();
-
-const profileEditValidator = new FormValidator(
-  consts.validatorConfig,
-  consts.profileEditDiv,
-);
+// initial loading handlers
 
-profileEditValidator.enableValidation();
+function handleProfileLoad({ name, about, avatar }) {
+  profileEl.setUserInfo(name, about);
+  profileEl.setAvatar(avatar);
+}
+function loadAll() {
+  api
+    .initialLoad()
+    .then((data) => {
+      handleProfileLoad(data[0]);
+      return data;
+    })
+    .then((data) => {
+      loadDefaultCards(data[1]);
+    });
+}
+function loadDefaultCards(cardsArr) {
+  if (cardsArr.length != 0) {
+    gallery["items"] = cardsArr;
+    gallery.renderItems();
+  } else {
+    console.log("Gallery empty. Loading default cards...");
+    gallery["items"] = addDefaultCards();
+    gallery.renderItems();
+  }
+}
+function addDefaultCards() {
+  const defaultCards = [];
+  consts.initialCards.forEach((card) => {
+    api.addCard(card);
+    defaultCards.push(card);
+  });
+  return defaultCards;
+}
 
+// validation
+const formValidation = [
+  new FormValidator(consts.validatorConfig, consts.addCardModalDiv),
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```